Matrixfile

```diff
@@ -153,7 +153,7 @@
     'resque2-redis4' => '✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ 3.2 / ✅ 3.3 / ✅ 3.4 / ✅ jruby'
   },
   'rest_client' => {
-    'http' => '✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ 3.2 / ✅ 3.3 / ✅ 3.4 / ✅ jruby'
+    'rest-client-latest' => '✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ 3.2 / ✅ 3.3 / ✅ 3.4 / ✅ jruby'
```

Comment:
Not sure which issue solves the renaming, but if there is an issue we may want to address it to @TonyCTHsu

Reply:
The idea is to remove dependency from the bloated `http` group, and allow automatic updates for each of those dependency:
https://github.com/DataDog/dd-trace-rb/pull/4424/files#diff-8dcc058cddb32b66124619c46f3d1df8d7e99ea9770ce9c0ddb120e68c360222L89

Reply:
and this actually reminds me that I should have added `rest-client` to `rake edge:update` task:
https://github.com/DataDog/dd-trace-rb/pull/4424/commits/5d335f3fef007c48a24b18c1d6550a31ad78d9fd

Reply:
That is correct!